pyproject.toml

```diff
@@ -1,15 +1,18 @@
 [tool.ruff]
+line-length = 88
+
+# NOTE(willkg): Need to keep using Python 3.8 until socorro-siggen drops
+# support for it
+target-version = "py38"
+
+[tool.ruff.lint]
```

Comment:
ruff adjusted configuration when it expanded into formatting, so this moves some options to their correct table.